plugin/trino-hive/src/main/java/io/trino/plugin/hive/LegacyHiveViewReader.java

```diff
@@ -45,7 +46,7 @@ public ConnectorViewDefinition decodeViewData(String viewData, Table table, Cata
                 translateHiveViewToTrino(viewText),
                 Optional.of(catalogName.toString()),
                 Optional.ofNullable(table.getDatabaseName()),
-                table.getDataColumns().stream()
+                Stream.concat(table.getDataColumns().stream(), table.getPartitionColumns().stream())
```

Comment:
Could you add a test to `AbstractTestHiveViews`?

Reply:
> Could you add a test to `AbstractTestHiveViews`?
Okay, I'm going to do that.